json/listmonk.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "listmonk",
+    "slug": "listmonk",
+    "categories": [
+        14
+    ],
+    "date_created": "2024-11-22",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": "9000",
+    "documentation": "https://listmonk.app/docs/",
+    "website": "https://listmonk.app/",
+    "logo": "https://listmonk.app/static/images/logo.svg",
+    "description": "High performance, self-hosted, newsletter and mailing list manager with a modern dashboard. Single binary app.",
+    "install_methods": [
+        {
+            "type": "default",
+            "script": "ct/listmonk.sh",
+            "resources": {
+                "cpu": "1",
+                "ram": "512",
+                "hdd": "4",
+                "os": "debian",
+                "version": "12"
+            }
+        }
+    ],
+    "default_credentials": {
```

Comment:
listmonk / listmonk?

Reply:
No, when you go to webpage after creation of LXC you can create a user